packages/manager/src/features/Databases/DatabaseDetail/DatabaseBackups/RestoreNewFromBackupDialog.tsx

```diff
@@ -0,0 +1,94 @@
+import { useSnackbar } from 'notistack';
+import * as React from 'react';
+import { useHistory } from 'react-router-dom';
+
+import { ActionsPanel } from 'src/components/ActionsPanel/ActionsPanel';
+import { Dialog } from 'src/components/Dialog/Dialog';
+import { Notice } from 'src/components/Notice/Notice';
+import { Typography } from 'src/components/Typography';
+import { useNewRestoreFromBackupMutation } from 'src/queries/databases/databases';
+import { getAPIErrorOrDefault } from 'src/utilities/errorUtils';
+
+import type { Database } from '@linode/api-v4/lib/databases';
+import type { DialogProps } from 'src/components/Dialog/Dialog';
+
+interface Props extends Omit<DialogProps, 'title'> {
+  database: Database;
+  onClose: () => void;
+  open: boolean;
+  restoreTime?: string;
+}
+
+export const RestoreNewFromBackupDialog: React.FC<Props> = (props) => {
```

Comment:
```suggestion
export const RestoreNewFromBackupDialog = (props: Props) => {
```
We can remove the React.FC here.